package.json

```diff
@@ -18,15 +18,16 @@
   "license": "MIT",
   "version": "4.0.0-beta.1",
   "files": [
-    "dist",
+    "vue2",
+    "vue3",
     "src"
   ],
-  "main": "dist/vue2/cjs",
-  "module": "dist/vue2/es",
+  "main": "vue2/cjs",
+  "module": "vue2/es",
```

Comment:
was already the case, but why not vue2/cjs/index.js ?

Reply:
any reason or concern not having `/index.js` in the path?

Reply:
I guess it's supported, but I assumed that module and main were files, not module identifiers. If it works, then there's probably no issue